src/mono/wasm/Wasm.Build.Tests/BuildTestBase.cs

```diff
@@ -1005,6 +1051,11 @@ internal BuildPaths GetBuildPaths(BuildArgs buildArgs, bool forPublish=true)
             return table;
         }
 
+        protected static string GetSkiaSharpReferenceItems()
+            => @"<PackageReference Include=""SkiaSharp"" Version=""2.88.3"" />
```

Comment:
Should the version number come from  something like `eng\Versions.props` ?

Reply:
This is specifically for the test here, and not used anywhere else. And now the version is set only in one location too. I don't think it should be in `eng/Versions.Props`.